decksite/data/deck.py

```diff
@@ -26,8 +26,9 @@ def load_deck(deck_id: int) -> Deck:
 
 def load_decks_count(where: str = 'TRUE',
                      having: str = 'TRUE',
-                     season_id: Optional[Union[str, int]] = None) -> int:
-    columns = 'COUNT(*) AS n'
+                     season_id: Optional[Union[str, int]] = None,
+                     distinct: bool = False) -> int:
```

Comment:
This should be unnecessary. Is there ever a time we call `load_decks_count` and we want to count decks more than once? You can just put the DISTINCT in every time.

Reply:
it's DISTINCT only if you join with a table which contains multiple entries per deck, which right now is only `deck_archetype_change`. Seems like an unnecessary overhead to add.

Reply:
My point is that `DISTINCT` always gets the correct result even when it's not strictly necessary. So we can do away with the cognitive overhead of the additional param at the expense of whatever it does to the query. But I'm also 90% convinced that it will not affect query speed more than the tiniest bit. So we can essentially do away with the cognitive overhead of the additional param at no cost.

Reply:
okay I made it always distinct